application/src/main/java/org/togetherjava/tjbot/commands/moderation/ModerationUtils.java

```diff
@@ -53,20 +51,18 @@ private ModerationUtils() {
     static final Color AMBIENT_COLOR = Color.decode("#895FE8");
 
     /**
-     * Provides you with an Enum of actions which have a timely constraint, like being muted for 1
-     * hour.
+     * Actions with timely constraint, like being muted for 1 hour.
      */
     private static final Set<ModerationAction> TEMPORARY_ACTIONS =
             EnumSet.of(ModerationAction.MUTE);
     /**
-     * Provides you with an Enum of actions which are revoking previously made actions on the user,
-     * like unmuting the user after it has been muted.
+     * Actions with revoking previously made actions on the user, like unmuting the user after it
+     * has been muted.
      */
     private static final Set<ModerationAction> REVOKE_ACTIONS =
             EnumSet.of(ModerationAction.UNMUTE, ModerationAction.UNQUARANTINE);
     /**
-     * Provides you with an Enum of actions which are letting you know of a violation you have
-     * committed.
+     * Actions which are letting you know of a violation you have committed.
```

Comment:
The important point for this is that its actions that still keep u in the guild (i.e. not kick and ban).

So maybe `Soft violations were the user still remains member of the guild, such as a warning`.